external/java/src/main/java/ch/epfl/dedis/lib/crypto/Bn256G2Point.java

```diff
@@ -0,0 +1,131 @@
+package ch.epfl.dedis.lib.crypto;
+
+import ch.epfl.dedis.lib.Hex;
+import com.google.protobuf.ByteString;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
I've had troubles with the sun standard libraries before. I think it's not in all the JDKs. Could you change it to https://docs.oracle.com/javase/8/docs/api/java/lang/UnsupportedOperationException.html ?

Reply:
Yeah, this makes problems on Android too. Thanks for fixing.